config/deploy.rb

```diff
@@ -43,5 +43,12 @@
       execute "cd #{current_path} && ./gradlew jar"
     end
   end
+
+  task :install_dist do
+    on roles(:app) do
+      execute "cd #{current_path} && ./gradlew build installDist"
+    end
+  end
 end
 after 'deploy:finished', 'gradle:jar'
+after 'deploy:finished', 'gradle:install_dist'
```

Comment:
would make sense to replace gradle:jar target, yes?  (new name is fine -- just don't know why we'd want to do the jar target if it's useless.